pkg/controller/elasticsearch/certificates/remoteca/reconcile.go

```diff
@@ -36,6 +36,7 @@ func Labels(esName string) client.MatchingLabels {
 func Reconcile(
 	c k8s.Client,
 	es esv1.Elasticsearch,
+	ca *certificates.CA,
```

Comment:
Let's not make this a pointer so we don't have to check for `nil` in the function.
(In practice we know it's not `nil` in this case (until further refactored :)) but it's not very explicit).